controller/cmd/service-mirror/cluster_watcher.go

```diff
@@ -26,14 +27,37 @@ type (
 	// it can be requeued up to N times, to ensure that the failure is not due to some temporary network
 	// problems or general glitch in the Matrix.
 	RemoteClusterServiceWatcher struct {
-		clusterName     string
-		clusterDomain   string
-		remoteAPIClient *k8s.API
-		localAPIClient  *k8s.API
-		stopper         chan struct{}
-		log             *logging.Entry
-		eventsQueue     workqueue.RateLimitingInterface
-		requeueLimit    int
+		clusterName        string
+		clusterDomain      string
+		remoteAPIClient    *k8s.API
+		localAPIClient     *k8s.API
+		stopper            chan struct{}
+		log                *logging.Entry
+		eventsQueue        workqueue.RateLimitingInterface
+		requeueLimit       int
+		probePort          int32
```

Comment:
I think these probe fields are unused and shouldn't be part of the RCSW.  I think we decided that the probe configuration should be loaded from the gateway annotations and stored on the gateway spec, instead of being per cluster.